.env.example

```
@@ -5,3 +5,12 @@ STRAVA_CLIENT_ID=your_client_id_here
 STRAVA_CLIENT_SECRET=your_client_secret_here
 STRAVA_CLIENT_ACCESS_TOKEN=your_access_token_here
 STRAVA_REDIRECT_URI=http://localhost:3000/api/strava/callback
+
+# Database Configuration
+# PostgreSQL connection settings
+
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_DB=regionriders
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_DB key should go before the POSTGRES_HOST key

(UnorderedKey)